app/oxide-rot-1/app.toml

```diff
@@ -9,7 +9,7 @@ version = 0
 
 [kernel]
 name = "oxide-rot-1"
-requires = {flash = 60544, ram = 2696}
+requires = {flash = 60644, ram = 2696}
```

Comment:
I think we went over by about 12 bytes but I bumped it by a little more to give us some room